beanstalk_worker/worker_dispatcher.py

```diff
@@ -56,3 +58,7 @@
 
     logger.debug("Deleting job: %s" % job_details)
     job.delete()
+
+    # remove handler from logging object
+    if 'dfh' in locals():
```

Comment:
we are not putting any debug log, did not get the point to initialize and remove the handler without writing anything.